ipa-core/src/net/mod.rs

```diff
@@ -17,6 +21,19 @@ pub use error::Error;
 pub use server::{MpcHelperServer, TracingSpanMaker};
 pub use transport::{HttpShardTransport, HttpTransport};
 
+static CRYPTO_INIT: Once = Once::new();
+
+/// Provides access to IPAs Crypto Provider (AWS Libcrypto).
+///
+/// # Panics
+/// If there are issues starting the default crypto provider
+pub fn get_crypto_provider() -> &'static Arc<CryptoProvider> {
+    CRYPTO_INIT.call_once(|| {
+        let _ = rustls::crypto::aws_lc_rs::default_provider().install_default();
+    });
+    CryptoProvider::get_default().expect("Default Crypto provider should be initialized")
+}
+
```

Comment:
```suggestion
/// Provides access to the default Crypto Provider used to setup Rustls for HTTP.
/// Currently, we use the default Rustls offering which is AWS libcrypto. 
pub fn get_crypto_provider() -> Arc<CryptoProvider> {
    Arc::new(rustls::crypto::aws_lc_rs::default_provider())
}
```

`install_default` still overrides the default global provider which is not what we want here I think.